README.md

```diff
@@ -27,13 +27,13 @@ composer require --dev wavevision/coding-standard
 Add to existing ruleset
 
 ```xml
-<rule ref="vendor/wavevision/coding-standard/php/WavevisionCodingStandard/ruleset.xml"/>
+<rule ref="vendor/wavevision/coding-standard/WavevisionCodingStandard"/>
```

Comment:
I understand the codesniffer code standard guidelines. The thing is, this package should still be maintained as a monorepo multi-language code style. I prefer keeping the `php` folder as it is in consistency with the rest of the structure. It is clear and well readable for everyone regardless they're using the coding standard as composer or npm dependency. Also, we're currently not maintaining any custom sniffs, so we don't need our own namespace.

Reply:
If you better not make this package a proper `phpcodesniffer-standard` then please close this PR.
This PR's sole purpose is that.

Reply:
Have you seen a monorepo with different language packages?

Reply:
BTW PHPCS is for PHP **and** JavaScript

> the main phpcs script that tokenizes PHP, JavaScript and CSS files to detect violations

Reply:
Thanks for your comments! I'll wait for @jfilla to share his thoughts with us.

> Have you seen a monorepo with different language packages?
 
Could you please provide a link to some package you know / use? It would be great to compare.

> BTW PHPCS is for PHP **and** JavaScript

Yes, I know. The fact is `eslint` and `prettier` prevail in this matter. We never intended to use `phpcs` for linting JavaScript.

Reply:
> > Have you seen a monorepo with different language packages?
> 
> Could you please provide a link to some package you know / use? It would be great to compare.

I barely know any monorepos.

Reply:
Same thing as above.